ramp4-pcar4/ramp4-pcar4
# Improve notification tray icons

### Related Item(s)
#2316 
#2311 

### Changes
- [FEATURE] Change notification tray `Clear all` and `Dismiss` (individual notification) button icons, to improve clarity.

### Notes
<img width="522" alt="image" src="https://github.com/user-attachments/assets/1f4a2c39-22bb-4a6d-8310-9859b7db6699">

### QA Testing

Please test against `main` branch https://ramp4-pcar4.github.io/ramp4-pcar4/main/demos/enhanced-samples.html

### Testing

Steps:
1. Go to classic sample 33 (or any other sample that reliably causes notifications).
2. Wait for notifications to show up, then click on the notification icon to open the tray.
3. See the new `Clear all` icon on the top bar and `Dismiss` icon for each notification.
4. Click dismiss for a notification, it should still work. Then, if possible, click clear all to remove all notifications.
